src/components/blocks/Rows/component.jsx

```diff
@@ -1,28 +1,17 @@
-import React, { useCallback } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import Row from '@/components/blocks/Row';
 
-const Rows = ({ rows, onRowDelete, onToggleDisable }) => {
-  const handleRowDelete = useCallback(
-    (id) => () => {
-      onRowDelete(id);
-    },
-    [onRowDelete],
-  );
-  const handleToggleDisable = useCallback(
-    (id) => () => {
-      onToggleDisable(id);
-    },
-    [onRowDelete],
-  );
-
+const Rows = ({ rows, onRowDelete, onToggleDisable, onChangeInput, onChangeSelect }) => {
   const renderRows = rows.map((row) => {
     return (
       <li key={row.id} className="list-group-item d-flex justify-content-between">
         <Row
           row={row}
-          onRowDelete={handleRowDelete(row.id)}
-          onToggleDisable={handleToggleDisable(row.id)}
+          onRowDelete={onRowDelete(row.id)}
+          onToggleDisable={onToggleDisable(row.id)}
+          onChangeInput={onChangeInput(row.id)}
+          onChangeSelect={onChangeSelect(row.id)}
```

Comment:
Этот код можно уменьшить если у тебя пропсы называются также как функции. Всё что тебе нужно просто пробросить их дальше вот так
```suggestion
         {...props}
```
А в месте где принимаешь пропсы не перечислять их все, а написать `...props`
`const Rows = ({ rows, onRowDelete, onToggleDisable, onChangeInput, onChangeSelect }) => {`
поменять на 
`const Rows = ({ rows, ...props }) => {`

* По желанию. Просто для уменьшения кода